core/primitives/src/errors.rs

```diff
@@ -487,8 +489,8 @@ pub enum ActionErrorKind {
     /// receipt validation.
     NewReceiptValidationError(ReceiptValidationError),
     /// Error occurs when a `CreateAccount` action is called on hex-characters
-    /// account of length 64.  See implicit account creation NEP:
-    /// <https://github.com/nearprotocol/NEPs/pull/71>.
+    /// account of length 64 or 42 (when starting with '0x').
+    /// See implicit account creation NEP: <https://github.com/nearprotocol/NEPs/pull/71>.
```

Comment:
nit: Rather than describing the format of either of the implicit account types can you just say something like:
"Error occurs when a `CreateAccount` action is called on a near-implicit or eth-implicit account." 
Also please reference the NEP for the EthImplicit as well.